pipettes/firmware/utility_gpio.c

```diff
@@ -120,7 +120,7 @@ void LED_drive_gpio_init() {
     HAL_GPIO_Init(hardware.port, &GPIO_InitStruct);
 }
 
-void sync_drive_gpio_init() {
+static void sync_drive_gpio_init() {
```

Comment:
What was the purpose of making these static?

Reply:
Same as making stuff `const` in cpp and other languages - in c, whatever functions can be static should be static. static applied to a function definition in c means that it's not accessible from other translation units, which means it doesn't pollute the global namespace